api/resource.rb

```diff
@@ -42,6 +42,12 @@ module Properties
       # GET/DELETE requests only.  In particular, this is often used
       # to add query parameters.
       attr_reader :self_link_query
+      # URL to be used to retrieve a list of these resources (for InSpec)
+      attr_reader :list_url
```

Comment:
I'd like the group's consensus on this.

Reply:
First thoughts: can't list_url default to the base_url? It seems to match in most every instance.

Reply:
Now that I think about it, how does `list_url` differ from `collection_url`?

Reply:
collection_url seems to be the same thing. Didn't see that before. It's probably worth keeping the query url because we need to know where to pull the collection from in the response?